addon/validators/ds-error.js

```diff
@@ -5,11 +5,11 @@
 
 import Ember from 'ember';
 import Base from 'ember-cp-validations/validators/base';
-const DS = self.DS;
+import require from 'require';
 
-if (typeof self.DS === 'undefined') {
-  throw new Error('Ember-Data is required to use the DS Error validator.');
-}
+const { assert } = Ember;
+const hasEmberData = require.has('ember-data');
```

Comment:
I would prefer to put this logic in [addon/utils/utils.js](https://github.com/offirgolan/ember-cp-validations/blob/master/addon/utils/utils.js#L9)